ci-operator/jobs/openshift/osd-network-verifier/openshift-osd-network-verifier-main-postsubmits.yaml

```diff
@@ -0,0 +1,54 @@
+postsubmits:
+  openshift/osd-network-verifier:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    cluster: build01
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 1
+    name: branch-ci-openshift-osd-network-verifier-main-publish-coverage
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-credentials-file=/etc/report/credentials
+        - --secret-dir=/secrets/certman-operator-codecov-token
+        - --target=publish-coverage
+        command:
+        - ci-operator
```

Comment:
just for knowledge: what does ci-operator do? How does it trigger coverage process?

Reply:
job files are generated by release jobs ( running `make update` before submitting the PR) so don't know precisely the internals :/ . I only updated myself the config file. 
But I need to update the secret names, I saw I didn't replace it properly before submitting :/

Reply:
ack, thanks